x/dex/types/keys.go

```diff
@@ -96,15 +111,19 @@ const (
 	ShortBookCountKey = "ShortBook-count-"
 )
 
-
+const (
```

Comment:
nit: why not merge the these two const closure as both of they hold constants for key

Reply:
:+1: